### ISSUE TITLE
NI thresholds for company directors

### ISSUE THREAD

**Issue by rasoolsomji** [PolicyEngine/policyengine-uk#721]
Hi there!

I really enjoy OpenFisca, but I recently encountered an issue.

### Here is what I did:
Tried to calculate the tax for a company director receiving a salary

### Here is what I expected to happen:
There to be an input to denote that you are a company director, in order to use the different national insurance thresholds that exist for directors

### Here is what actually happened:
I could find no input on the website, nor in a quick scan, could I find it in this repo

### Here is data (or links to it) that can help you reproduce this issue:
https://www.gov.uk/employee-directors
https://www.gov.uk/guidance/rates-and-thresholds-for-employers-2022-to-2023 (Ctrl+F 'director')

In short, company directors, in 2022-23, pay 12.73% NI (because their salary is averaged over the whole year) on earnings over £11,908

This is not an uncommon situation, since many contractors and freelancers operate as one-person Ltd companies paying themselves a mixture of salary and dividends

**Comment by vahid-ahmadi** [comment]
We don't have the company director National Insurance thresholds in our codebase so far. @MaxGhenis Can I close it?

**Comment by MaxGhenis** [comment]
If it's still an enhancement opportunity please leave it open
